Vagrantfile

```diff
@@ -1,159 +1,92 @@
-  # some recipes and/or roles.
-  #
-  # config.vm.provision "chef_solo" do |chef|
-  #   chef.cookbooks_path = "../my-recipes/cookbooks"
-  #   chef.roles_path = "../my-recipes/roles"
-  #   chef.data_bags_path = "../my-recipes/data_bags"
-  #   chef.add_recipe "mysql"
-  #   chef.add_role "web"
-  #
-  #   # You may also specify custom JSON attributes:
-  #   chef.json = { mysql_password: "foo" }
-  # end
-
-  # Enable provisioning with chef server, specifying the chef server URL,
-  # and the path to the validation key (relative to this Vagrantfile).
-  #
-  # The Opscode Platform uses HTTPS. Substitute your organization for
-  # ORGNAME in the URL and validation key.
-  #
-  # If you have your own Chef Server, use the appropriate URL, which may be
-  # HTTP instead of HTTPS depending on your configuration. Also change the
-  # validation key to validation.pem.
-  #
-  # config.vm.provision "chef_client" do |chef|
-  #   chef.chef_server_url = "https://api.opscode.com/organizations/ORGNAME"
-  #   chef.validation_key_path = "ORGNAME-validator.pem"
-  # end
-  #
-  # If you're using the Opscode platform, your validator client is
-  # ORGNAME-validator, replacing ORGNAME with your organization name.
-  #
-  # If you have your own Chef Server, the default validation client name is
-  # chef-validator, unless you changed the configuration.
-  #
-  #   chef.validation_client_name = "ORGNAME-validator"
-  config.vm.provision "shell", path: "scripts/provision.sh", run: "always"
-
-  config.vm.post_up_message = %{
-  #######################################################################
-
-  Things should be set up for you to test MPTCP from the virtual machine.
-  To validate that MPTCP is working from inside the vm, issue
-    vagrant ssh -c \"curl www.multipath-tcp.org\"
-
-  You should get a joyful message announcing you are using MPTCP.
-
-  To log in, just issue the command
-    vagrant ssh
-
-  You can now become root with 
-    sudo su
-
-  and manage this debian virtual machine.
-
-   #######################################################################
-}   
-
-  config.trigger.after :up do 
-    run "./scripts/post_up.sh"
-  end
-  config.trigger.after :halt do 
-    run "./scripts/post_halt.sh"
-  end
-end
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+    # The most common configuration options are documented and commented below.
+    # For a complete reference, please see the online documentation at
+    # https://docs.vagrantup.com.
+  
+    # Every Vagrant development environment requires a box. You can search for
+    # boxes at https://vagrantcloud.com/search.
+    config.vm.define :client do |client|
+      client.vm.box = "fedora/38-cloud-base"
```

Comment:
Fedora 38 is already EoL :)

Is it easy to upgrade?

Reply:
oh seriously ? Time to upgrade my OS then :'(
For the changes, just modifying 38-cloud-base by 40-cloud-base should do the job :D

Reply:
>  For the changes, just modifying 38-cloud-base by 40-cloud-base should do the job :D

Can you switch to Fedora 40 then please?

(or switch to a LTS distribution: Debian, Ubuntu, RHEL, RockyLinux, etc. ;) )